### ISSUE TITLE
Server kicks me out.

### ISSUE THREAD

**Issue by hddnninja** [SuperMartijn642/SimpleMagnets#9]
My server kicks me out when I activate the magnet inside my Bauble slot. I can not rejoin. I am on 1.12.2

[FML]: Exception caught during firing event net.minecraftforge.fml.common.gameevent.TickEvent$PlayerTickEvent@6aa4ca96:
20.12 15:48:41 [Server] INFO java.lang.NoSuchMethodError: net.minecraft.util.math.AxisAlignedBB.<init>(Lnet/minecraft/util/math/Vec3d;Lnet/minecraft/util/math/Vec3d;)V
20.12 15:48:41 [Server] INFO at com.supermartijn642.simplemagnets.MagnetItem.func_77663_a(MagnetItem.java:61) ~[MagnetItem.class:?]
20.12 15:48:41 [Server] INFO at com.supermartijn642.simplemagnets.integration.BaublesActive$1.onWornTick(BaublesActive.java:34) ~[BaublesActive$1.class:?]
20.12 15:48:41 [Server] INFO at baubles.common.event.EventHandlerEntity.playerTick(EventHandlerEntity.java:96) ~[EventHandlerEntity.class:?]
20.12 15:48:41 [Server] INFO at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_517_EventHandlerEntity_playerTick_PlayerTickEvent.invoke(.dynamic) ~[?:?]
20.12 15:48:41 [Server] INFO at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90) ~[ASMEventHandler.class:?]
20.12 15:48:41 [Server] INFO at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182) [EventBus.class:?]
20.12 15:48:41 [Server] INFO at net.minecraftforge.fml.common.FMLCommonHandler.onPlayerPostTick(FMLCommonHandler.java:370) [FMLCommonHandler.class:?]
20.12 15:48:41 [Server] INFO at net.minecraft.entity.player.EntityPlayer.func_184808_cD(EntityPlayer.java:378) [aed.class:?]
20.12 15:48:41 [Server] INFO at net.minecraft.entity.player.EntityPlayer.func_70071_h_(EntityPlayer.java:288) [aed.class:?]
20.12 15:48:41 [Server] INFO at net.minecraft.entity.player.EntityPlayerMP.func_71127_g(EntityPlayerMP.java:382) [oq.class:?]
20.12 15:48:41 [Server] INFO at net.minecraft.network.NetHandlerPlayServer.func_73660_a(NetHandlerPlayServer.java:173) [pa.class:?]
20.12 15:48:41 [Server] INFO at net.minecraftforge.fml.common.network.handshake.NetworkDispatcher$1.func_73660_a(NetworkDispatcher.java:209) [NetworkDispatcher$1.class:?]
20.12 15:48:41 [Server] INFO at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:285) [gw.class:?]
20.12 15:48:41 [Server] INFO at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:180) [oz.class:?]
20.12 15:48:41 [Server] INFO at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:790) [MinecraftServer.class:?]
20.12 15:48:41 [Server] INFO at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:397) [nz.class:?]
20.12 15:48:41 [Server] INFO at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668) [MinecraftServer.class:?]
20.12 15:48:41 [Server] INFO at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526) [MinecraftServer.class:?]
20.12 15:48:41 [Server] INFO at java.lang.Thread.run(Thread.java:748) [?:1.8.0_265]
20.12 15:48:41 [Server] Server thread/ERROR [FML]: Index: 7 Listeners:
20.12 15:48:41 [Server] Server thread/ERROR [FML]: 0: NORMAL
20.12 15:48:41 [Server] Server thread/ERROR [FML]: 1: ASM: class com.feed_the_beast.ftblib.FTBLibEventHandler onPlayerTick(Lnet/minecraftforge/fml/common/gameevent/TickEvent$PlayerTickEvent;)V
20.12 15:48:41 [Server] Server thread/ERROR [FML]: 2: ASM: class com.buuz135.industrial.item.infinity.OneThreeFiveHandler onPlayerTick(Lnet/minecraftforge/fml/common/gameevent/TickEvent$PlayerTickEvent;)V
20.12 15:48:41 [Server] Server thread/ERROR [FML]: 3: ASM: class chumbanotz.mutantbeasts.EventHandler onPlayerUpdate(Lnet/minecraftforge/fml/common/gameevent/TickEvent$PlayerTickEvent;)V
20.12 15:48:41 [Server] Server thread/ERROR [FML]: 4: ASM: class com.oblivioussp.spartanshields.event.EventHandlerCommon onPlayerTick(Lnet/minecraftforge/fml/common/gameevent/TickEvent$PlayerTickEvent;)V
20.12 15:48:41 [Server] Server thread/ERROR [FML]: 5: ASM: vazkii.quark.tweaks.feature.CompassesWorkEverywhere@7770b7db onUpdate(Lnet/minecraftforge/fml/common/gameevent/TickEvent$PlayerTickEvent;)V
20.12 15:48:41 [Server] Server thread/ERROR [FML]: 6: ASM: vazkii.quark.misc.feature.MapMarkers@66f55bd0 tick(Lnet/minecraftforge/fml/common/gameevent/TickEvent$PlayerTickEvent;)V
20.12 15:48:41 [Server] Server thread/ERROR [FML]: 7: ASM: baubles.common.event.EventHandlerEntity@7c4762c4 playerTick(Lnet/minecraftforge/fml/common/gameevent/TickEvent$PlayerTickEvent;)V
20.12 15:48:41 [Server] Server thread/ERROR [FML]: 8: ASM: class cursedflames.bountifulbaubles.event.EventHandler playerTick(Lnet/minecraftforge/fml/common/gameevent/TickEvent$PlayerTickEvent;)V
20.12 15:48:41 [Server] Server thread/ERROR [FML]: 9: ASM: svenhjol.charm.brewing.feature.EnderSight@2d76ef5e onPlayerTick(Lnet/minecraftforge/fml/common/gameevent/TickEvent$PlayerTickEvent;)V
20.12 15:48:41 [Server] Server thread/ERROR [FML]: 10: ASM: svenhjol.charm.world.feature.SpectreHaunting@408396bb onTick(Lnet/minecraftforge/fml/common/gameevent/TickEvent$PlayerTickEvent;)V
20.12 15:48:41 [Server] Server thread/ERROR [FML]: 11: ASM: lain.mods.cos.InventoryManager@540c0f1a handleEvent(Lnet/minecraftforge/fml/common/gameevent/TickEvent$PlayerTickEvent;)V
20.12 15:48:41 [Server] Server thread/ERROR [FML]: 12: ASM: com.brandon3055.draconicevolution.handlers.CustomArmorHandler@8491dd7 onPlayerTick(Lnet/minecraftforge/fml/common/gameevent/TickEvent$PlayerTickEvent;)V
20.12 15:48:41 [Server] Server thread/ERROR [FML]: 13: ASM: INSTANCE onPlayerTick(Lnet/minecraftforge/fml/common/gameevent/TickEvent$PlayerTickEvent;)V
20.12 15:48:41 [Server] Server thread/ERROR [FML]: 14: ASM: ivorius.reccomplex.events.handlers.RCForgeEventHandler@4bd19420 onPlayerTick(Lnet/minecraftforge/fml/common/gameevent/TickEvent$PlayerTickEvent;)V
20.12 15:48:41 [Server] Server thread/ERROR [FML]: 15: ASM: c4.comforts.common.capability.CapabilitySleeping$EventHandler@386a0485 onPostPlayerTick(Lnet/minecraftforge/fml/common/gameevent/TickEvent$PlayerTickEvent;)V
20.12 15:48:41 [Server] Server thread/ERROR [FML]: 16: ASM: zairus.megaloot.util.MLEventHandler@4f2c1781 onPlayerTick(Lnet/minecraftforge/fml/common/gameevent/TickEvent$PlayerTickEvent;)V
20.12 15:48:41 [Server] Server thread/ERROR [FML]: 17: ASM: net.blay09.mods.netherportalfix.NetherPortalFix@7b629e5c onPlayerTick(Lnet/minecraftforge/fml/common/gameevent/TickEvent$PlayerTickEvent;)V
20.12 15:48:41 [Server] Server thread/ERROR [FML]: 18: ASM: openblocks.common.ServerTickHandler@5df0f3c4 onPlayerTick(Lnet/minecraftforge/fml/common/gameevent/TickEvent$PlayerTickEvent;)V
20.12 15:48:41 [Server] Server thread/ERROR [FML]: 19: ASM: lumien.randomthings.handler.RTEventHandler@267e3f05 tick(Lnet/minecraftforge/fml/common/gameevent/TickEvent;)V
20.12 15:48:41 [Server] Server thread/ERROR [FML]: 20: ASM: ivorius.reccomplex.events.handlers.RCForgeEventHandler@4bd19420 onTick(Lnet/minecraftforge/fml/common/gameevent/TickEvent;)V
20.12 15:48:41 [Server] Server thread/ERROR [FML]: 21: ASM: com.sk89q.worldedit.forge.ThreadSafeCache@263a207b tickStart(Lnet/minecraftforge/fml/common/gameevent/TickEvent;)V
20.12 15:48:41 [Server] Server thread/ERROR [FML]: 22: ASM: com.creativemd.creativecore.common.event.CreativeTickHandler@532c2a68 onEventTick(Lnet/minecraftforge/fml/common/gameevent/TickEvent;)V
20.12 15:48:41 [Server] Server thread/WARN [minecraft/NetworkSystem]: Failed to handle packet for /98.230.178.216:52061
20.12 15:48:41 [Server] INFO net.minecraft.util.ReportedException: Ticking player
20.12 15:48:41 [Server] INFO at net.minecraft.entity.player.EntityPlayerMP.func_71127_g(EntityPlayerMP.java:459) ~[oq.class:?]
20.12 15:48:41 [Server] INFO at net.minecraft.network.NetHandlerPlayServer.func_73660_a(NetHandlerPlayServer.java:173) ~[pa.class:?]
20.12 15:48:41 [Server] INFO at net.minecraftforge.fml.common.network.handshake.NetworkDispatcher$1.func_73660_a(NetworkDispatcher.java:209) ~[NetworkDispatcher$1.class:?]
20.12 15:48:41 [Server] INFO at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:285) ~[gw.class:?]
20.12 15:48:41 [Server] INFO at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:180) [oz.class:?]
20.12 15:48:41 [Server] INFO at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:790) [MinecraftServer.class:?]
20.12 15:48:41 [Server] INFO at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:397) [nz.class:?]
20.12 15:48:41 [Server] INFO at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668) [MinecraftServer.class:?]
20.12 15:48:41 [Server] INFO at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526) [MinecraftServer.class:?]
20.12 15:48:41 [Server] INFO at java.lang.Thread.run(Thread.java:748) [?:1.8.0_265]
20.12 15:48:41 [Server] INFO Caused by: java.lang.NoSuchMethodError: net.minecraft.util.math.AxisAlignedBB.<init>(Lnet/minecraft/util/math/Vec3d;Lnet/minecraft/util/math/Vec3d;)V
20.12 15:48:41 [Server] INFO at com.supermartijn642.simplemagnets.MagnetItem.func_77663_a(MagnetItem.java:61) ~[MagnetItem.class:?]
20.12 15:48:41 [Server] INFO at com.supermartijn642.simplemagnets.integration.BaublesActive$1.onWornTick(BaublesActive.java:34) ~[BaublesActive$1.class:?]
20.12 15:48:41 [Server] INFO at baubles.common.event.EventHandlerEntity.playerTick(EventHandlerEntity.java:96) ~[EventHandlerEntity.class:?]
20.12 15:48:41 [Server] INFO at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_517_EventHandlerEntity_playerTick_PlayerTickEvent.invoke(.dynamic) ~[?:?]
20.12 15:48:41 [Server] INFO at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90) ~[ASMEventHandler.class:?]
20.12 15:48:41 [Server] INFO at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182) ~[EventBus.class:?]
20.12 15:48:41 [Server] INFO at net.minecraftforge.fml.common.FMLCommonHandler.onPlayerPostTick(FMLCommonHandler.java:370) ~[FMLCommonHandler.class:?]
20.12 15:48:41 [Server] INFO at net.minecraft.entity.player.EntityPlayer.func_184808_cD(EntityPlayer.java:378) ~[aed.class:?]
20.12 15:48:41 [Server] INFO at net.minecraft.entity.player.EntityPlayer.func_70071_h_(EntityPlayer.java:288) ~[aed.class:?]
20.12 15:48:41 [Server] INFO at net.minecraft.entity.player.EntityPlayerMP.func_71127_g(EntityPlayerMP.java:382) ~[oq.class:?]
20.12 15:48:41 [Server] INFO ... 9 more
20.12 15:48:41 [Disconnect] User HddnNinja has disconnected, reason: Internal server error
20.12 15:48:41 [Server] Server thread/INFO [minecraft/DedicatedServer]: HddnNinja left the game

**Comment by SuperMartijn642** [comment]
It is fixed now in version 1.0.8b for Minecraft 1.12.

Thanks for reporting the issue!

**Comment by epicacrylic** [comment]
Hey, saw this issue and thought I'd ask if this is the same problem that I'm having?

Minecraft 1.12.2
Simple Magnets 1.1.11

```[03:38:17] [Server thread/WARN] [net.minecraft.network.NetworkSystem]: Failed to handle packet for /47.188.164.207:55230
net.minecraft.util.ReportedException: Ticking player
	at net.minecraft.entity.player.EntityPlayerMP.func_71127_g(EntityPlayerMP.java:459) ~[oq.class:?]
	at net.minecraft.network.NetHandlerPlayServer.func_73660_a(NetHandlerPlayServer.java:173) ~[pa.class:?]
	at net.minecraftforge.fml.common.network.handshake.NetworkDispatcher$1.func_73660_a(NetworkDispatcher.java:209) ~[NetworkDispatcher$1.class:?]
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:285) ~[gw.class:?]
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:180) [oz.class:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:790) [MinecraftServer.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:397) [nz.class:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668) [MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526) [MinecraftServer.class:?]
	at java.lang.Thread.run(Thread.java:750) [?:1.8.0_422]
Caused by: java.lang.NoSuchMethodError: net.minecraft.util.math.AxisAlignedBB.<init>(Lnet/minecraft/util/math/Vec3d;Lnet/minecraft/util/math/Vec3d;)V
	at com.supermartijn642.simplemagnets.MagnetItem.func_77663_a(MagnetItem.java:71) ~[MagnetItem.class:?]
	at net.minecraft.item.ItemStack.func_77945_a(ItemStack.java:526) ~[aip.class:?]
	at net.minecraft.entity.player.InventoryPlayer.func_70429_k(InventoryPlayer.java:363) ~[aec.class:?]
	at net.minecraft.entity.player.EntityPlayer.func_70636_d(EntityPlayer.java:511) ~[aed.class:?]
	at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:2179) ~[vp.class:?]
	at net.minecraft.entity.player.EntityPlayer.func_70071_h_(EntityPlayer.java:234) ~[aed.class:?]
	at net.minecraft.entity.player.EntityPlayerMP.func_71127_g(EntityPlayerMP.java:382) ~[oq.class:?]
	... 9 more
[03:38:17] [Server thread/INFO] [net.minecraft.network.NetHandlerPlayServer]: Epic_Acrylic lost connection: Internal server error
[03:38:17] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer]: Epic_Acrylic left the game
```

It's specifically when I activate it, and it prevents me from being able to be on the server unless I edit my NBT data to set "activated" to 0 again

I normally use this mod in my 1.19/1.20 packs so I figured it would be nice to have it on this 1.12 pack but this error is stopping that lol

Thank you for your time regardless ^^
